src/Eyes/Domain/DomainStatus.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace BeholderWebClient\Eyes\Domain;
+
+use BeholderWebClient\Eyes\Status;
+
+class DomainStatus extends Status {
+
+  const REQUERIMENT_FAIL = 'REQUERIMENT FAIL: whois command not found with type -P whois';
```

Comment:
whois command not found with type -P whois
I think it's not clear what that means.
Perhaps you should do:
Whois command not found.  Tried to execute: "type -P whois"